python/tests/test_problem.py

```diff
@@ -585,8 +585,8 @@ def test_problem_limits(self):
             self.assertTupleEqual(l.shape,(net.num_buses,))
             self.assertTupleEqual(u.shape,(net.num_buses,))
             for bus in net.buses:
-                self.assertEqual(bus.v_max,u[bus.index_v_mag])
-                self.assertEqual(bus.v_min,l[bus.index_v_mag])
+                self.assertEqual(bus.v_max_reg,u[bus.index_v_mag])
+                self.assertEqual(bus.v_min_reg,l[bus.index_v_mag])
```

Comment:
This test is for checking variable limits. As they have nothing to do with voltage regulation, they should not be "reg" limits. In this test, v_max_reg and v_min_reg should be changed to norm limits or he simpler aliases and the C functions that create the limit vectors u and l need to be changed as well ( should be mentioned in a comment below(